Src/CSharpier/CSharpier.csproj

```diff
@@ -28,4 +28,8 @@
         <ProjectReference Include="..\CSharpier.Generators\CSharpier.Generators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
     </ItemGroup>
 
+    <Target Name="husky" BeforeTargets="Restore;CollectPackageReferences" Condition="'$(HUSKY)' != 0">
```

Comment:
Didn't you say you were going to make these opt-in for this repo?

Reply:
I also updated the husky json file because it was not obvious the first part of it was a comment.